data/human/human missions.txt

```diff
@@ -5733,3 +5733,227 @@ mission "Argosy Hijacking (Declined)"
 		fleet "Small Syndicate"
 		to spawn
 			has "Argosy Hijacking: declined"
+
+
+
+mission "A Dead Man's Path 1"
+	name "Siblings to New China"
+	description "Bring these siblings to New China to hear their father's last will by <date>. They offer to pay you <payment>."
+	minor
+	passengers 2
+	deadline 7
+	source
+		near "Merak" 3 6
+		not attributes "station"
+	destination
+		planet "New China"
+	to offer
+		random < 20
+	on offer
+		conversation
+			`The spaceport on <origin> is lively tonight. There are porters running to and from depots with crates, crewmen buying rounds of drinks for each other around the bars before they hit the bed, and bands of Navy officers jovially greeting captains and systematically inspecting ships. The energy and self-purpose of this spaceport is on full display this one night.`
+			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch them drift from one person to another, each time being sent off without an answer. Finally, they come towards you.`
+			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
+			choice
+				`	"Yes, my name's Captain <last>."`
+				`	"Depends on what you want."`
+					goto suspicious
+			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to <destination>. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course."`
+				goto helpornot
+			label suspicious
+			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`
+			label helpornot
+			choice
+				`	"Of course I can."`
+					goto agreed
+				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
+			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
```

Comment:
> with both heads looking down

Not a huge fan of this.

Reply:
Why not? The next best option I can think of is:
```suggestion
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, their heads down.`
```
but it seems like you'd like to avoid that sort of expression?

Reply:
Yeah. It's probably fine though.